scripts/update/Updater.php

```diff
@@ -747,5 +747,26 @@ public function update($initialVersion) {
 
             $this->setVersion('5.32.0');
         }
+
+        if ($this->isVersion('5.32.0')) {
+
+            $extension = \common_ext_ExtensionsManager::singleton()->getExtensionById('taoQtiTest');
+
+            $config = $extension->getConfig('testRunner');
+            if (!isset($config['shortcuts'])) {
+                $config['shortcuts'] = [];
+            }
```

Comment:
Not sure this check is relevant here, as the previous update should have set the `shortcuts` array entry. Anyway, this is not so important.